lib/storage/InstanceSync/index.native.js

```diff
@@ -0,0 +1,6 @@
+/**
+ * Noop on native
```

Comment:
Can you please add a little more to the comment do explain why it's a noop (because this feature is only used to keep multiple browser tabs in sync)?